models/backup.go

```diff
@@ -57,3 +57,11 @@ type BackupResult struct {
 	Resources        []string
 	IgnoredResources []string
 }
+
+var (
+	// generic configurations
+	ConfigTTL              = "ttl"
+	ConfigIgnoreDownstream = "ignore_downstream"
+
+	BackupSpecKeyURN = "URN"
+)
```

Comment:
this could be `const` and defined at the top of the file?

Reply:
Updated